### DISCUSSION TITLE
Events-all

### DISCUSSION THREAD

**Discussion by nagydavid** [zachowj/node-red-contrib-home-assistant-websocket#788] [Q&A]
Hi 
I have a question regarding the events: all node.

How do specify the filter for event data? 

Can you provide me an example?
<img width="575" alt="image" src="https://user-images.githubusercontent.com/10994218/204522337-9d5f682c-456a-4982-bd7c-8122a0f9f5ba.png">

**Comment by zachowj** [comment]
Use a debug node after the events: all node and it's a whole/partial  JSON object of what is in `msg.payload.event`.
